src/mame/video/starfield_05xx.h

```diff
@@ -0,0 +1,44 @@
+// license:BSD-3-Clause
+// copyright-holders:Robert Hildinger
+#ifndef MAME_VIDEO_STARFIELD_05XX_H
+#define MAME_VIDEO_STARFIELD_05XX_H
+
+#pragma once
+
+#include "screen.h"
```

Comment:
Why is this here?  Unnecessary includes increase dependencies.  I can't see anything that actually depends on it.

Reply:
Change committed